DNN Platform/Library/Entities/Users/UserInfo.cs

```diff
@@ -114,6 +114,36 @@ public string FirstName
         [Browsable(false)]
         public bool IsSuperUser { get; set; }
 
+
+        /// <summary>
+        /// Gets whether the user is in the portal's administrators role
+        /// </summary>
+        private bool? isAdmin;
+        public bool IsAdmin
```

Comment:
I need to check a few things on the processes, but I believe that IsInRole already handles the IsSuperUser check as a superuser is always in every role.